src/main/java/com/example/demo/repository/TeamRepository.java

```diff
@@ -0,0 +1,59 @@
+package com.example.demo.repository;
+
+import com.example.demo.model.Team;
+import com.example.demo.tournament.CSVcontroller;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class TeamRepository implements BaseTeamRepository {
+
+    private static final List<Team> teams;
+
+    static {
+        teams = new ArrayList<>();
+        CSVcontroller.Read(teams);
+    }
+
+    @Override
+    public List<Team> getList() {
+        return teams;
+    }
+
+    @Override
+    public boolean add(Team team) {
+        teams.add(team);
+        return true;
+    }
+
+    public void readFromFile() {
+        CSVcontroller.Read(teams);
+    }
+
+    public void writeToFile() {
+        CSVcontroller.Write(teams);
+    }
+
+    public void print() {
```

Comment:
It's ok for debugging, but if this code is used to print teams into console for UI than it should be in different component. This one should only care about storage of teams